File: Week2/fizzbuzzkata/fizzbuzzkata.py
def fizzbuzz(num):
    if num == 1:
        return "1"
    elif num == 2:
        return "2"
    elif num%5 == 0 and num%3 ==0:
        return "FizzBuzz"
    elif num == 3:
        return "Fizz"
    elif num == 5:
        return "Buzz"
    elif num % 3 == 0:
        return "Fizz"
    elif num % 5 == 0:
        return "Buzz"
    else:
        return ""


def main():
    for num in range(1,16):
        print(fizzbuzz(num))

main()

File: Week6/fizzbuzzkata/Fizzbuzzkata.js
module.exports = {
    fizzBuzz: function(val){
        if(val == 1){
            return "1";
        }
        else if(val == 2){
            return "2";
        }
        else if((val % 3 == 0) && (val % 5 ==0)){
            return "FizzBuzz"
        }
        else if(val%3==0){
            return "Fizz"
        }
        else if(val % 5==0){
            return "Buzz"
        }
        
       
    }
    
}
